sdk/resourcemanager/azure-resourcemanager-search/src/test/java/SearchTests.java

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.azure.resourcemanager.search.SearchServiceManager;
+import com.azure.resourcemanager.search.models.SearchService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class SearchTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private SearchServiceManager searchServiceManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        searchServiceManager = SearchServiceManager
+            .configure().withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure().withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withSubscription("faa080af-c1d8-40ad-9cce-e1a450ca5b57");
```

Comment:
Just with `withDefaultSubscription()`, which uses the env variable (a hardcoded id may not work if dev run it locally on their different subscription).

Reply:
Fixed in the new version.